client-spark/spark-2/src/main/java/org/apache/spark/shuffle/celeborn/RssShuffleManager.java

```diff
@@ -172,6 +172,7 @@ public <K, V> ShuffleWriter<K, V> getWriter(
               "Unrecognized shuffle write mode!" + celebornConf.shuffleWriterMode());
         }
       } else {
+        sortShuffleIds.add(handle.shuffleId);
```

Comment:
seems we have to add parenthesis after shuffleId
```
handle.shuffleId()
```

Reply:
thanks, fixed